src/components/CoolFrame.vue

```diff
@@ -79,14 +86,32 @@ export default {
 			}
 		})
 	},
+	beforeDestroy() {
+		window.removeEventListener('message', this.handlePostMessage)
+	},
+	methods: {
+		handlePostMessage(event) {
+			try {
+				const data = event.data
+				if (data.MessageId === 'Iframe_Height') {
+					document.getElementById(this.iframeName).height = data.Values.ContentHeight
+				}
+			} catch (e) {
+				console.error('Something went wrong with post message', e)
+			}
+		},
+	},
+
 }
 </script>
 
   <style scoped>
   .cool-frame-iframe {
     width: 100%;
     border: none;
-	height: 2000px;
-	overflow-y: auto;
+	overflow-y: hidden;
```

Comment:
Should we still overflow auto just to be sure that if something goes wrong users are still able to reach all elements in the iframe?